meta-lmp-bsp/dynamic-layers/xilinx/recipes-bsp/u-boot/u-boot-xlnx/k26/lmp-base.cfg

```diff
@@ -0,0 +1,42 @@
+CONFIG_DEFAULT_DEVICE_TREE="zynqmp-zcu102-rev1.0"
+CONFIG_XILINX_PS_INIT_FILE="psu_init_gpl.c"
+CONFIG_BUILD_TARGET="u-boot.bin"
+CONFIG_SPL_CRYPTO_SUPPORT=y
+CONFIG_SPL_HASH_SUPPORT=y
+CONFIG_SPL_FIT_PRINT=y
+CONFIG_FIT_SIGNATURE=y
+CONFIG_SPL_FIT_SIGNATURE=y
+CONFIG_SPL_RSA_VERIFY=y
+CONFIG_RSA=y
+CONFIG_ENV_IS_NOWHERE=n
+CONFIG_ENV_IS_IN_FAT=y
+CONFIG_ENV_FAT_INTERFACE="mmc"
+CONFIG_ENV_FAT_DEVICE_AND_PART="0:1"
+CONFIG_ENV_OFFSET=0x1440000
```

Comment:
I've chosen this offsets based on custom layout I used for boot image on QSPI on Avnet Board.
These values might overlap with something else (from https://xilinx-wiki.atlassian.net/wiki/spaces/A/pages/1641152513/Kria+K26+SOM#Boot-FW-QSPI-Memory-Map, it's `“Image B” - BOOT.BIN`)


`),

Reply:
I think these two configs for ENV offsets/size can be just dropped for now

Reply:
Indeed, will update this one.